certification/internal/policy/container/base_on_ubi.go

```diff
@@ -25,7 +26,7 @@ func NewBasedOnUbiCheck(layerHashChecker layerHashChecker) *BasedOnUBICheck {
 }
 
 func (p *BasedOnUBICheck) Validate(ctx context.Context, imgRef certification.ImageReference) (bool, error) {
-	layerHashes, err := p.getImageLayers(ctx, imgRef.ImageInfo)
```

Comment:
Since this particular one is an internal call, it made sense to just remove context for now. If we need it in the future, it's at least not an exported API.